doctordroid/README.md

```diff
@@ -0,0 +1,23 @@
+# Doctor Droid
+
+## [](https://github.com/DataDog/integrations-extras/tree/master/doctordroid#overview)Overview
+
+Doctor Droid is an observability analytics platform. This integration allows Datadog users to analyse their alerts history to decide better thresholds for alerts, identify alerts coverage of their services and helps the user fetch metrics from upstream and downstream services to enrich their alerts for faster diagnosis.
+
+## [](https://github.com/DataDog/integrations-extras/tree/master/doctordroid#setup)Setup
+
+### [](https://github.com/DataDog/integrations-extras/tree/master/doctordroid#installation)Installation
+Visit [Doctor Droid](https://alertops-app.drdroid.io/)  to sign up for free. Once registered, visit the [integrations page](https://alertops-app.drdroid.io/integrations)  and add a Datadog integration. This guides you through the Datadog OAUTH2 flow to grant Doctor Droid access to your API Keys to query APM & Infra metrics for you.
```

Comment:
Users should kick off authorization from the Datadog integrations page, rather than from Doctor Droid. When they start authorization from within Datadog, their site is returned to you as a query parameter that you can use to construct API calls. Would you be able to update this accordingly?

Reply:
Hi @anawishnoff

The authrozation kicks off from datadog integrations page only. If the user already has an account with dr droid, the flow proceeds to oauth step. If not then user is prompted to first sign up on dr droid platform and then is redirected to datadog auth step.

We are following a strategy similar to the vantage integration. Please let me know if there are any changes needed here.

Reply:
Hi @droid-mohit , just to be clear, are the following steps correct? 
1. User goes to Dr Droid integration tile in Datadog
2. User goes to Configure tab and clicks "Connect Accounts"
3. User is taken to the Integrations page within Doctor Droid
4. User adds the integration in Doctor Droid and is guided through the OAuth flow

Reply:
Yes, these are the steps followed. The OAuth flow takes places within datadog.